security/certificate_types_descriptions/etcd-certificates.adoc

```diff
@@ -0,0 +1,32 @@
+[id="cert-types-etcd-certificates"]
+= etcd certificates
+include::modules/common-attributes.adoc[]
+:context: cert-types-etcd-certificates
+
+toc::[]
+
+== Purpose
+
+etcd certificates are signed by the etcd-signer; they come from a certificate authority (CA) that is generated by the bootstrap process.
+
+== Expiration
+
+The CA certificates are valid for 10 years. The peer, client, and server certificates are valid for three years.
+
+== Management
+
+These certificates are managed by the system and not the user.
+
+== Services
+
+etcd certificates are used for encrypted communication between etcd member peers, as well as encrypted client traffic. The following certificates are generated and used by etcd and other processes that communicate with etcd:
+
+* Peer certificates: Used for communication between etcd members.
+* Client certificates: Used for encrypted server-client communication. Client certificates are currently used by the API server only, and no other service should connect to etcd directly except for the proxy. Client secrets (`etcd-client`, `etcd-metric-client`, `etcd-metric-signer`, and `etcd-signer`) are added to the `openshift-config`, `openshift-monitoring`, and `openshift-kube-apiserver` namespaces.
+* Server certificates: Used by the etcd server for authenticating client requests.
+* Metric certificates: All metric consumers connect to proxy with metric-client certificates.
+
+.[discrete]
```

Comment:
Remove the . before [discrete]; this is causing an error in rendering that heading.